internal/monitors/collectd/redis/redis.go

```diff
@@ -86,6 +86,14 @@ func (c *Config) PythonConfig() *python.Config {
 	return c.pyConf
 }
 
+func (c *Config) sendListLengthsTuples() [][]interface{} {
+	var out [][]interface{}
+	for _, ll := range c.SendListLengths {
+		out = append(out, []interface{}{ll.DBIndex, ll.KeyPattern})
```

Comment:
Remind me how yaml list elements databaseIndex and keyPattern get mapped to struct fields DBIndex and KeyPattern.

Reply:
Through the struct tags `yaml:"..."` on the config struct.

Reply:
SendListLengths []ListLength `yaml:"sendListLengths"`?!

Reply:
Yes, that is it.

Reply:
Ok, I'm sure it works. I just don't see how databaseIndex gets mapped to DBIndex unless it by position. We'll have chat about it when we meet.